### ISSUE TITLE
Graphics window seems to be "broken"

### ISSUE THREAD

**Issue by sneshyba** [arose/nglview#915]
I'm using jupyter on ubuntu (via jupyterhub), a notebook with %matplotlib notebook selected. And while  nglview.show_structure_file is trying to do something, all get is this broken graphics window. Sorry, I'm new to this, not sure if the image below will paste properly.

![image](https://user-images.githubusercontent.com/1376118/87582580-7132d080-c68f-11ea-9a09-a0d287d95ef6.png)

**Comment by hainm** [comment]
hi @sneshyba, you need to `display` the 3D viewer before rendering it.

```
# cell 1
view = ...
view # live

# cell 2
view.render_image()

# cell 3
view._display_image() # just for displaying in static notebook. 
```

Please check this example: https://github.com/arose/nglview/raw/master/examples/images/membrane.gif

**Comment by sneshyba** [comment]
Yes, I had done that before. The results are similar, but still not rendered:

![image](https://user-images.githubusercontent.com/1376118/87589262-cd9aed80-c699-11ea-8e1e-85b676e1aa38.png)

![image](https://user-images.githubusercontent.com/1376118/87589309-df7c9080-c699-11ea-8de6-f5198574622e.png)

**Comment by hainm** [comment]
@sneshyba did you follow the install guide here? https://github.com/arose/nglview#released-version

If you did, please read further here: https://github.com/arose/nglview/blob/master/docs/FAQ.md#widget-not-shown

**Comment by sneshyba** [comment]
I did -- but then I re-checked and realized that enabling nglview had given a permission error. So I used sudo and that solved it! 

jupyter-nbextension enable nglview --py --sys-prefix (gave a permission error)
sudo jupyter-nbextension enable nglview --py --sys-prefix (worked)

Thank you so much! I'm really liking MDAnalysis, especially since my students are not able to get in front of a workstation easily -- now they can do a lot more remotely.

I still have to figure out how to display my .gro files properly -- but I think that's a different ticket.
